Frontend/research/src/Actions/Index.js

```diff
@@ -55,7 +55,7 @@ export const GET_LINKS_FAILURE = "GET_LINKS_FAILURE";
 
 export const getLinks = id => dispatch => {
     dispatch({ type: GET_LINKS })
-    axios.get(`https://rticle.herokuapp.com/api/user/${id}`, {headers: {Authorization: localStorage.getItem('token')}})
+    axios.get(`https://rticle.herokuapp.com/api/user/${id}`, { headers: { Authorization: localStorage.getItem('token') } })
```

Comment:
you may find it beneficial to make the url into a constant and then use that in your calls. this way if you ever need to change it, you only need to change it in one place. i.e. const URL = https://some-url.here  then in your axios calls it would be like axios.get(`${URL}/api/user/${id}`, {headers........}) something like this, and if you need to use it in oher files you can export the const that you created and import it anywhere you may need it